.prettierrc.json

```diff
@@ -5,5 +5,6 @@
 	"singleQuote": true,
 	"trailingComma": "es5",
 	"useTabs": true,
-	"tabWidth": 4
+	"tabWidth": 4,
+	"printWidth": 120
```

Comment:
setting as 120 chars as prettier is make some _pretty_ awful formatting decisions. But happy to remove if anyone feels strongly

Reply:
We have big screens, set it to something longer, please

Reply:
Tbh, i think 120 is a good compromise, especially for viewing on things like Github. Can we leave it at 120 and increase if necessary?